docs/features/logforwarding/outputs/splunk-forwarding.md

```diff
@@ -0,0 +1,66 @@
+
```

Comment:
Please convert to adoc in case docs wants to consume some version of it

Reply:
Will do.